modules/ROOT/pages/metrics/custom-metrics-connector.adoc

```diff
@@ -0,0 +1,96 @@
+= Anypoint Custom Metrics Connector
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using Custom Dashboards. 
```

Comment:
Please say what you mean by hybrid or at least provide a link to the Marketing hybrid site. Most people seeing hybrid think cars.